<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autoref</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitterV">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitterTop">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="DebugTreeWidget" name="debugTree">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitterBottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="LogWidget" name="log">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="FieldWidget" name="field">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionSidesFlipped"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Logging</string>
    </property>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionConfiguration"/>
    <addaction name="actionShowOptions"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menuVideo"/>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockVisualization">
   <property name="windowTitle">
    <string>Visualizations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="VisualizationWidget" name="visualization"/>
  </widget>
  <widget class="QDockWidget" name="dockTiming">
   <property name="windowTitle">
    <string>Timing</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="TimingWidget" name="timing"/>
  </widget>
  <widget class="QDockWidget" name="dockAutoref">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Autoref</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="AutorefTeamWidget" name="autoref"/>
  </widget>
  <widget class="QDockWidget" name="dockOptions">
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="OptionsWidget" name="options"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSidesFlipped">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Flip sides</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Record file</string>
   </property>
   <property name="toolTip">
    <string>Record File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionShowOptions">
   <property name="text">
    <string>Toggle options panel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutorefTeamWidget</class>
   <extends>QWidget</extends>
   <header>autorefteamwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisualizationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/visualizationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/timingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FieldWidget</class>
   <extends>QGraphicsView</extends>
   <header>widgets/fieldwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/logwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DebugTreeWidget</class>
   <extends>QTreeView</extends>
   <header>widgets/debugtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>OptionsWidget</class>
   <extends>QWidget</extends>
   <header>optionswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
